packages/config/src/memory/memory.config.ts

```diff
@@ -77,7 +74,7 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
   assets: string;
 
   /** Catch configs with the same imagery that using the different imagery ids. */
-  duplicateImagery: ConfigTileSet[] = [];
+  duplicateImagery: string[] = [];
```

Comment:
I think we need more than just the ids how about `{ name: string, id: string, projection: number}[]` ?

Reply:
Looks good to me.

Reply:
Updated.